
All tests work for me, but I had to make a few changes to the
test script and Makefile to get it working. These changes would not
have been in the files I sent you earlier. Here are the diffs:

*** db.1.85/test/Makefile       Sun Aug 28 11:42:52 1994
--- db.1.85/test/Makefile.irix  Wed Mar  1 11:52:47 1995
***************
*** 9,15 ****
  INC=  -I${PORTDIR}/include -I${PORTDIR}
  OORG= -g
  #STAT=        -DSTATISTICS
! CFLAGS=       -D__DBINTERFACE_PRIVATE -DDEBUG ${STAT} ${OORG} ${INC}
  
  dbtest: ${OBJS} ${PORTDIR}/libdb.a
        ${CC} -o $@ ${OBJS} ${PORTDIR}/libdb.a
--- 9,15 ----
  INC=  -I${PORTDIR}/include -I${PORTDIR}
  OORG= -g
  #STAT=        -DSTATISTICS
! CFLAGS=       -D__DBINTERFACE_PRIVATE -DDEBUG ${STAT} ${OORG} ${INC} -D__BIT_TYPES_DEFINED__
  
  dbtest: ${OBJS} ${PORTDIR}/libdb.a
        ${CC} -o $@ ${OBJS} ${PORTDIR}/libdb.a

*** db.1.85/test/run.test       Thu Sep  1 18:04:44 1994
--- db.1.85/test/run.test.irix  Mon Oct 16 12:43:41 1995
***************
*** 16,21 ****
--- 16,23 ----
                DICT=/usr/share/dict/words
        elif [ -f /usr/dict/words ]; then
                DICT=/usr/dict/words
+       elif [ -f /usr/share/lib/dict/words ]; then
+               DICT=/usr/share/lib/dict/words
        else
                echo 'run.test: no dictionary'
                exit 1

